content/docs/components/hyperparameter-tuning/experiment.md

```diff
@@ -350,18 +350,24 @@ To define the metrics collector for your experiment:
       containing a 
       [tf.Event](https://www.tensorflow.org/api_docs/python/tf/Event). You
       should specify the path in the `source` field.
+    * `Custom`: Specify this value if you need to use custom way to collect
+      metrics, and you need to define your custom metrics collector container
```

Comment:
Make two sentences instead of one, to clarify that the second part is a consequence of the first part:

"Specify this value if you need to use custom way to collect metrics. You must define your custom metrics collector container"